scripts/compile_all_examples.py

```diff
@@ -204,11 +183,21 @@ def checked_compile(
     return CompilationResult(
         rel_path=rel_path,
         ok=ok,
-        teal_files=[root / p for p in teal_files_written],
+        bin_files=[root / p for p in bin_files_written],
         stdout=result.stdout if not ok else "",  # don't thunk stdout if no errors
     )
 
 
+def _load_template_vars(path: Path) -> Iterable[str]:
+    if path.exists():
+        for line in path.read_text().splitlines():
+            if line.startswith("prefix="):
+                prefix = line.strip("prefix=")
```

Comment:
this strips all `p,r,e,f,i,x,=` characters - so `prefix=fixme` becomes `me` for example - you want `.removeprefix`